tests/run/inlinecode.pyx

```diff
@@ -14,4 +14,4 @@ def inline_langversion(language_level, a=5, b=2):
     """
     # Caching for inline code didn't always respect language version.
     # https://github.com/cython/cython/issues/3419
-    print(Shadow.inline(_inline_divcode, language_level=language_level, quiet=True)(a=a, b=b))
+    print(cython.inline(_inline_divcode, language_level=language_level, quiet=True)(a=a, b=b))
```

Comment:
IIRC, I tried `cython.inline` at first, but it gets interpreted [as the C `inline` modifier](https://cython.readthedocs.io/en/latest/src/tutorial/pure.html#extension-types-and-cdef-functions) when the file gets compiled as a `.pyx` test.

Reply:
Hmm, right. I think this should actually go into `TestInline.py` and not a `.pyx` file.